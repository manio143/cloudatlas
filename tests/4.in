SELECT first(99, name) AS new_contacts ORDER BY cpu_usage DESC NULLS LAST, num_cores ASC NULLS FIRST
