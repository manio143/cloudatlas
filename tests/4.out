/uw: new_contacts: [khaki13, violet07, khaki31]
/pjwstk: new_contacts: [whatever01, whatever02]
/: new_contacts: [uw, pjwstk]
